### PR TITLE
Add Client Bindings page to left-navigation

### PR DESCRIPTION
**Pull request by bmbouter**
https://pulp.plan.io/issues/5723
closes #5723

### CODE REVIEW

File: docs/glossary.rst [hunk 1]
```diff
@@ -46,9 +46,8 @@ Glossary
         :term:`type` of :term:`content unit<content>`.
 
     pulpcore
-        A generalized backend with a Plugin API and a :doc:`REST
-        API<integration-guide/index>`. It uses :term:`plugins<plugin>` to manage
-        :term:`content`.
+        A python package offering users :doc:`client_bindings` and plugin writers a
+        :ref:`Plugin API`. It is :term:`plugin` based and manages :term:`content`.
```
Comment (daviddavis) [inline docs/glossary.rst]: Maybe we should keep REST API and link it to the rest_api page?

Comment (bmbouter) [reply docs/glossary.rst]: Oh yeah I did this incorrectly. I had meant to say REST API. Let me fix.

Comment (bmbouter) [reply docs/glossary.rst]: I had meant to make this REST API. Let me fix this.

Comment (bmbouter) [reply docs/glossary.rst]: I pushed the version linking to the REST and Pugin APIs.

**Pull request merged**